catalog/spatial/registry/registry-rest-endpoint/src/main/java/org/codice/ddf/registry/rest/endpoint/report/RegistryReportHelper.java

```diff
@@ -58,14 +41,7 @@
 import oasis.names.tc.ebxml_regrep.xsd.rim._3.SlotType1;
 import oasis.names.tc.ebxml_regrep.xsd.rim._3.TelephoneNumberType;
 
-/**
- * Generates a registry report for a given registry metacard.
- * Displays the Name, General, Collections,
- * Services, Organizations, and Contacts information for the node.
- */
-@Path("/")
-public class RegistryReportViewer {
-    private static final Logger LOGGER = LoggerFactory.getLogger(RegistryReportViewer.class);
+public class RegistryReportHelper {
```

Comment:
Can we name this class something else than `Helper`? Using _Helper_ in class name is considered a code smell and usually indicates that the class' responsibilities are unclear.